src/HonzaBotner.Discord.Services/Commands/BotCommands.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using HonzaBotner.Discord.Services.Attributes;
+using HonzaBotner.Discord.Services.Options;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace HonzaBotner.Discord.Services.Commands
+{
+    [Group("bot")]
+    [Description("Commands to get you info about Discord bot and other things.")]
+    public class BotCommands : BaseCommandModule
+    {
+        private readonly ILogger<BotCommands> _logger;
+        private readonly InfoOptions _infoOptions;
+
+        public BotCommands(ILogger<BotCommands> logger, IOptions<InfoOptions> infoOptions)
+        {
+            _logger = logger;
+            _infoOptions = infoOptions.Value;
+        }
+
+        [Command("activity")]
+        [Description("Changes bot's activity status.")]
+        [RequireMod]
+        public async Task Activity(CommandContext ctx,
+            [Description("Type of the activity. Allowed values: competing, playing, streaming, watching or listening.")]
+            string type,
```

Comment:
```suggestion
            ActivityType type,
```
We can use enum directly here